R/table-object.R

```diff
@@ -885,6 +899,16 @@ st_mutate <- function(x, ...) {
   x
 }
 
+dplyr_filter <- dplyr::filter
```

Comment:
Just out of curiosity: why do this assignment rather than call `dplyr::filter` directly?  The only thing I could come up with was avoiding the `::` lookup overhead for each `st_filter` call, but that seems too minor to be the reason.

Reply:
Yeah it's not performance but coding style. I still get nervous about `filter()` but I'm seeing other usage and this should find `dplyr::filter()`.  I'll revert this and trust the lookup.

Reply:
Just realizing that filter is in stats namespace; so using it unqualified after importing from `dplyr` (and not `stats`) should be safe. Thanks for picking this up.